<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>848</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>737</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="tools_box">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="save_image"/>
    <addaction name="open"/>
    <addaction name="new_scene"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вигляд</string>
    </property>
    <addaction name="tools"/>
    <addaction name="console"/>
    <addaction name="settings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_scene"/>
   <addaction name="separator"/>
   <addaction name="open"/>
   <addaction name="separator"/>
   <addaction name="save"/>
   <addaction name="separator"/>
   <addaction name="save_image"/>
   <addaction name="separator"/>
  </widget>
  <action name="open">
   <property name="icon">
    <iconset>
     <normaloff>folder-horizontal-open.png</normaloff>folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Відкрити</string>
   </property>
  </action>
  <action name="save_as">
   <property name="icon">
    <iconset>
     <normaloff>disks-black.png</normaloff>disks-black.png</iconset>
   </property>
   <property name="text">
    <string>Зберегти як...</string>
   </property>
  </action>
  <action name="tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>hammer.png</normaloff>hammer.png</iconset>
   </property>
   <property name="text">
    <string>Інструменти</string>
   </property>
  </action>
  <action name="console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>terminal.gif</normaloff>terminal.gif</iconset>
   </property>
   <property name="text">
    <string>Консоль</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset>
     <normaloff>disk-black.png</normaloff>disk-black.png</iconset>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="new_scene">
   <property name="icon">
    <iconset>
     <normaloff>plus-circle.png</normaloff>plus-circle.png</iconset>
   </property>
   <property name="text">
    <string>Нова сцена</string>
   </property>
  </action>
  <action name="save_image">
   <property name="icon">
    <iconset>
     <normaloff>image.png</normaloff>image.png</iconset>
   </property>
   <property name="text">
    <string>Зберегти зображення...</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset>
     <normaloff>gear.png</normaloff>gear.png</iconset>
   </property>
   <property name="text">
    <string>Налаштування</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
